api/src/main/kotlin/pl/edu/pk/backend/app/InitDatabase.kt

```diff
@@ -0,0 +1,57 @@
+package pl.edu.pk.backend.app
+
+import io.vertx.core.Future
+import io.vertx.core.json.JsonObject
+import pl.edu.pk.backend.Services
+import pl.edu.pk.backend.model.Role
+import pl.edu.pk.backend.model.User
+
+class InitDatabase(val services: Services) {
+
+  fun initDatabase() {
+    if (Config.profile != "local") return
+    services.userService.getUsers().onSuccess {
+      if (it.isEmpty()) {
+        addUsers().compose {
+          addTickets()
+        }
+      }
+    }
+  }
+
+  fun addUsers(): Future<User> {
+    return services.userService.createUser("Tomek", "Nowak", "tomek@tomek.pl", "123456")
+      .compose {
+        services.userService.patchUser("tomek@tomek.pl", "Tomek",
```

Comment:
For the patch method we can define default value of optional arguments as null. Then you can use named parameters and just write:
```
services.userService.patchUser("tomek@tomek.pl", "Tomek",  roles = listOf(Role.Admin, Role.User))
```
There are more cases like this. in the ticket patch method too.